### DISCUSSION TITLE
Governance team KYC process for LDN applications

### DISCUSSION THREAD

**Discussion by raghavrmadya** [filecoin-project/notary-governance#606] [General]
For the clarity of the community, I'm outlining the KYC process undertaken by the governance team for Fil+ applications. **Please note that this does not reflect the due diligence process, which is undertaken by the notaries**

1. Completeness - This means that answers are provided for all fields in the application. There is a bot to assess this stage and if the application passes this stage, the "bot:lookingGood" label kicks-in. **Please note that the bot does not always look for what exactly the content is and there are times when the client does not replace the placeholder text or writes jibberish **

2. Once an application is scanned for completeness by the bot, a human (member of the governance team, community member, notary) can ask questions to clarify anything stated in the application. 

**There is no rule of what to ask, how many questions to ask**
3. A member of the governance team makes an effort to ensure things such as but not limited to - 

- The client is not faking to be someone else.
- Actually needs the DataCap amount and limit requested
- Has confidence that the client with distributing with SPs fairly
- The data samples are aligned with what is being described in the application
- Data samples reflect the need for what is being requested. For example - If an application asks for 5 PiBs and has 5 .jpeg files in the sample, that is not sufficient to gain the confidence of a governance team member.

4. If a governance team member, after reviewing all aspects of the application, including questions and engagement by community members, has some confidence that the client is who they say they are, a trigger is pushed and notaries are invited to conduct due diligence, propose a DataCap request, and a separate notary must approve the request for the first tranche of DataCap to go to the address provided by the client.

**Comment by raghavrmadya** [comment]
As a member of the governance team, I do not follow a playbook or template while conducting KYC as it is easy to hack and emulate. I make the best possible human effort that can range from IP address searches, looking for on-chain behavior on filfox.io and the interplanetary dashboard, and often asking members of the client growth team if the client has been in-touch at a ecosystem event, over phone/email or another means. This is often the case when a client is asking for high than the current DataCap limit. As such, the way I gain confidence is multi-faceted and if it was as simple as following a list of 10 items, we would not need a human to go through applications and would have already automated it. That being said, if you disagree with a trigger, you are welcome to add your comment on the application itself to alert notaries before they propose and approve an application.

**Comment by raghavrmadya** [comment]
It is also important to mention that we ARE making efforts to automate some aspects of this in a way that leads to increased visibility of trust such as risk scores. Some examples of this were presented in governance calls such as Fil+ Canvas and Fil+ Watch. 

Community members are welcome to propose innovative ways in which we can bring more automation while maintaining transparency in the aforementioned process

**Reply by goldthwaiteeverette61** [reply]
A suggestion: For example, if my website is https://www.abc.com, then visit https://www.abc.com/f1xxxxxxxx and return to the f1xxxxxxxx (On-chain address for first allocation) that submitted the LDN application.

**Reply by goldthwaiteeverette61** [reply]
Prove the site is mine

**Reply by Destore2023** [reply]
This only suits individual applicants, how can a company like Amazone or Mircosoft make their website like: www.aws.com/f1xxxxxx or www.mircosoft.com/f1xxxxxx

> A suggestion: For example, if my website is https://www.abc.com, then visit https://www.abc.com/f1xxxxxxxx and return to the f1xxxxxxxx (On-chain address for first allocation) that submitted the LDN application.

**Comment by tqlzpg** [comment]
已收到您的来信，我会尽快处理
